### PR TITLE
feature/647

### LINKED ISSUE #647: New Airport: KTPA (Tampa Intl.)
Tampa International Airport (IATA: TPA, ICAO: KTPA, FAA LID: TPA) is a public airport six miles west of Downtown Tampa, in Hillsborough County, Florida, United States.

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #647 

### Completion Checklist
- [x] terrain file (empty)
- [x] polygonal airspace boundary
- [x] all SIDs
- [x] all STARs
- [x] information to validate traffic levels
- [x] base video map
- [x] no console errors or warnings

---

According to Wikipedia, KTPA has 189,865 yearly operations. Traffic level should fall between 22acph and 65acph. Currently, there's a total of 26.5acph 👍 

Blocked by #662.

### CODE REVIEW

File: assets/airports/ktpa.json [hunk 1]
```diff
@@ -0,0 +1,1566 @@
...
+                ["SZW", "HEVVN", "LEGGT", "TABIR", "DARBS", "PIE"]
+            ]
+        },
+        "DEAKK5": {
+            "icao": "DEAKK5",
+            "name": "Deak Five",
+            "entryPoints": {
+                "LBV": ["LBV", "ROGAN"],
+                "PHK": ["PHK", "ROGAN"],
+                "RSW": ["RSW", "ROGAN"]
```
Comment (erikquinn) [inline assets/airports/ktpa.json]: `ROGAN` can be moved to the body

File: assets/airports/ktpa.json [hunk 2]
```diff
@@ -0,0 +1,1566 @@
...
+                ["N28.15830194", "W83.20832944"],
+                ["N28.27375250", "W83.10793389"],
+                ["N28.30531639", "W83.08094972"],
+                ["N28.55532333", "W82.86782472"]
+            ]
+        }
+    ],
+    "fixes": {
+        "_1L187PIE116": ["N27.848198", "W82.546200"],
+        "_1R187PIE116": ["N27.842546", "W82.533100"],
```
Comment (erikquinn) [inline assets/airports/ktpa.json]: `_RWY1LPIE116` would probably be more consistent with the other forms we've used.

File: assets/airports/ktpa.json [hunk 3]
```diff
@@ -0,0 +1,1566 @@
...
+        "SYKES": ["N27d39.88", "W83d02.99"],
+        "SZW"  : ["N30d33.37", "W84d22.44"],
+        "TABIR": ["N28d25.19", "W83d00.20"],
+        "TAY"  : ["N30d30.28", "W82d33.17"],
+        "TUMPY": ["N28d10.58", "W82d25.78"],
+        "WILON": ["N29d25.07", "W82d41.59"],
+        "WOBKI": ["N27d41.89", "W82d37.61"]
+    },
+    "restricted": [
+
```
Comment (erikquinn) [inline assets/airports/ktpa.json]: please remove empty line

File: assets/airports/ktpa.json [hunk 4]
```diff
@@ -0,0 +1,1566 @@
...
+            ["N28d28m20.478", "W82d13m54.175", "N28d28m20.478", "W82d13m54.175"],
+            ["N28d28m07.739", "W82d08m25.818", "N28d28m07.377", "W82d08m14.613"],
+            ["N28d28m07.377", "W82d08m14.613", "N28d28m07.377", "W82d08m14.613"],
+            ["N28d28m05.231", "W82d07m20.075", "N28d28m04.867", "W82d07m08.867"],
+            ["N28d28m04.867", "W82d07m08.867", "N28d28m04.867", "W82d07m08.867"],
+            ["N28d27m51.531", "W82d01m40.542", "N28d27m51.158", "W82d01m29.337"],
+            ["N28d27m51.158", "W82d01m29.337", "N28d27m51.158", "W82d01m29.337"],
+            ["N28d27m48.639", "W82d00m34.806", "N28d27m48.266", "W82d00m23.599"],
+            ["N28d27m48.266", "W82d00m23.599", "N28d27m48.266", "W82d00m23.599"],
+            ["N28d29m47.172", "W82d21m56.734", "N28d29m37.267", "W
```
Comment (erikquinn) [inline assets/airports/ktpa.json]: this line has a bunch of spaces at the end, please remove

File: assets/airports/airportLoadList.js [hunk 5]
```diff
@@ -304,6 +304,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             premium: false,
             wip: true
         },
+        {
+            icao: 'ktpa',
+            level: 'easy',
+            name: 'Tampa International Airport',
+            wip: false
```
Comment (erikquinn) [inline assets/airports/airportLoadList.js]: please add `premium: false` here as well.

File: assets/airports/ktpa.json (hunk 2 above)
Comment (Fechulo) [reply assets/airports/ktpa.json]: done 👍

File: assets/airports/ktpa.json (hunk 3 above)
Comment (Fechulo) [reply assets/airports/ktpa.json]: Forgot to remove it when I realised there were no restricted areas inside the TRACON :man_facepalming: fixed 👍

File: assets/airports/ktpa.json (hunk 1 above)
Comment (Fechulo) [reply assets/airports/ktpa.json]: done 👍

File: assets/airports/ktpa.json (hunk 4 above)
Comment (Fechulo) [reply assets/airports/ktpa.json]: done 👍

File: assets/airports/airportLoadList.js (hunk 5 above)
Comment (Fechulo) [reply assets/airports/airportLoadList.js]: done 👍

**Comment by erikquinn** [comment]
Work complete on #662. Unblocking.

**Pull request merged**